hack/jenkins/build_iso.sh

```diff
@@ -101,5 +101,8 @@ else
 	git remote add minikube-bot git@github.com:minikube-bot/minikube.git
 	git push -f minikube-bot ${branch}
 
-	gh pr create --fill --base master --head minikube-bot:${branch}
+	pr=$(gh pr create --fill --base master --head minikube-bot:${branch} \
+	      | grep -o -E "github.com/kubernetes/minikube/pull/[0-9]+" \
```

Comment:
what is the extra unrelated changes meant for ?

Reply:
what do you mean? we get PR number on lines 104-105 parsing `gh pr create` output and then adding `/ok-to-test` comment on line 107. All changes are important here.